.idea/misc.xml

```diff
@@ -5,31 +5,37 @@
     <option name="myDefaultNotNull" value="android.support.annotation.NonNull" />
```

Comment:
You might wanna add these extra files to .gitIgnore

Reply:
Ok, I'm sorry, this is my first time I am contributing to something.
Do I add this to the .gitIgnore and then I should recommit my changes?